src/domains/comments/models/@types/comments.typedef.js

```diff
@@ -0,0 +1,51 @@
+import {gql} from 'apollo-server';
+
+const commentTypeDefs = gql`
+  type Comment {
+    _id: ID!
+    comment: String!
+    author: String!
+    ipv4: String!
+    isDeleted: Boolean
+    createdAt: DateTime!
+    updatedAt: DateTime!
+    deletedAt: DateTime
+    isChild: Boolean!
+    postId: Post!
+    replies: [Comment]
+  }
+
+  extend type Query {
+    comment(_id: ID!): Comment
+    comments(postId: ID!): [Comment]
+    replies(_id: ID!): Comment
+  }
+
+  extend type Mutation {
+    addComments(
+      comment: String!
+      author: String!
+      postId: ID!
+    ): Comment
```

Comment:
요런식의 방법도 있습니당
```suggestion
  input CommentInput {
      comment: String!
      author: String!
      postId: ID!
  }

  extend type Mutation {
    addComments(comment: CommentInput!): Comment
```

요런식으로 하면, input 값이 객체로 관리될 수 있어서 좀 더 깔끔할거 같아요~! (인자가 많아질 경우나, 반복된 인자를 계속 사용할 때)

https://graphql.org/graphql-js/mutations-and-input-types/

Reply:
사용하는 법은 GraphQL 객체 타입을 쓰는거랑 거의 동일합니당

Reply:
@mango906 좋은 정보 넘나 감사합니다. 4680c1b 커밋에 반영하였습니다~